examples/generator/mock_to_qir.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+
+# Copyright(c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import sys
+from antlr4 import *
+from argparse import *
+from mock_language.MockLanguageLexer import MockLanguageLexer
+from mock_language.MockLanguageParser import MockLanguageParser
+from mock_language.MockLanguageListener import MockLanguageListener
+from pathlib import Path
+from pyqir_generator import QirBuilder
+
+
+class QirGenerator(MockLanguageListener):
+    """
+    Class that generates QIR when walking the parse tree 
+    of a Mock language program.
+    """    
+    
+    def __init__(self, nr_qubits: int, module_id: str):
+        """
+        :param nr_qubits: The total number of qubits used in the compilation.
+        :type nr_qubits: int
```

Comment:
This is strictly speaking redundant with the type hint at line 22; most IDEs and docgen tools should be able to look at the type hint rather than relying on this RST directive.
```suggestion
```